include/eve/module/core/regular/popcount.hpp

```diff
@@ -7,10 +7,24 @@
 #pragma once
 
 #include <eve/arch.hpp>
-#include <eve/detail/overload.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
+#include <eve/detail/assert_utils.hpp>
 
 namespace eve
 {
+  template<typename Options>
+  struct popcount_t : elementwise_callable<popcount_t, Options>
+  {
+    template<integral_value T>
```

Comment:
You didnt changed the concept on the implementation so callign with a signed integer will lead to mapping.

Reply:
Also the implementation in SVE and NEON are non consistent.
Either you fix them or you go back to unsigned_value here.

Reply:
I fixed sve and x86, but there is nothing in neon, so did you mean x86?

Reply:
Yes x86. My issue is that it is not fixed and it only works because we do test on unsigned only.
In SVE, svcnt_x requires unsigned integer types yet the concept is integer not unsigned_integer.
If you want to keep it this way, hwich different from what STD does, you need a bit_cast in the implementation. I thnik we should go back to popcoutn only working on unsigned values and remove the concept in the implemenation.